ui/src/main/java/com/xmartlabs/bigbang/ui/common/recyclerview/SimpleItemRecycleItemType.java

```diff
@@ -2,9 +2,16 @@
 
 import android.support.annotation.NonNull;
 
+/**
+ * Simple {@link RecycleItemType} implementation with an empty {@code onBindViewHolder} implementation.
+ *
+ * @param <I>  The item related to the {@link android.support.v7.widget.RecyclerView.ViewHolder}.
+ * @param <VH> The {@link android.support.v7.widget.RecyclerView.ViewHolder} related to the item.
```

Comment:
instead of related to the item, i would put something like: `that holds the item visually`?